src/SharpBrick.PoweredUp/Hubs/Hub_Properties.cs

```diff
@@ -77,8 +77,8 @@ private async Task InitialHubPropertiesQueryAsync()
                 RequestHubPropertySingleUpdate(HubProperty.LegoWirelessProtocolVersion),
                 RequestHubPropertySingleUpdate(HubProperty.SystemTypeId),
                 RequestHubPropertySingleUpdate(HubProperty.HardwareNetworkId),
-                RequestHubPropertySingleUpdate(HubProperty.PrimaryMacAddress),
-                RequestHubPropertySingleUpdate(HubProperty.SecondaryMacAddress)
+                RequestHubPropertySingleUpdate(HubProperty.PrimaryMacAddress)
+                //RequestHubPropertySingleUpdate(HubProperty.SecondaryMacAddress) does not work on the TwoPortHandset
```

Comment:
I will make this better. Wait for a PR

Reply:
Done: #121 Please review the other PR so i can merge that to master for you to merge here.